scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SMALL_INSTALL=false
+
+case "${1:-}" in
+    -h|--help)
+        echo -e "Usage:\n\tquick-helm-install.sh [options]"
+        echo " "
+        echo "Installs StackRox via Helm charts, requires Helm."
+        echo " "
+        echo "options:"
+        echo "-h, --help                show brief help"
+        echo "-s, --small               reduce StackRox resource requirements for small clusters"
+        exit 0
+        ;;
+    -s|--small)
+        SMALL_INSTALL=true
+        ;;
+esac
+
+echo "Adding the stackrox/helm-charts/opensource repository to Helm."
+
+helm repo add stackrox https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+
+echo "Generating STACKROX_ADMIN_PASSWORD"
+
+STACKROX_ADMIN_PASSWORD="$(openssl rand -base64 20 | tr -d '/=+')"
+
+echo "Installing stackrox-central-services"
+
+helm install -n stackrox --create-namespace stackrox-central-services stackrox/stackrox-central-services --set central.adminPassword.value="${STACKROX_ADMIN_PASSWORD}"
+
+if $SMALL_INSTALL; then
+    echo "Upgrading stackrox to a small install"
+    helm upgrade -n stackrox stackrox-central-services stackrox/stackrox-central-services \
+      --set central.resources.requests.memory=1Gi \
+      --set central.resources.requests.cpu=1 \
+      --set central.resources.limits.memory=4Gi \
+      --set central.resources.limits.cpu=1 \
+      --set scanner.autoscaling.disable=true \
+      --set scanner.replicas=1 \
+      --set scanner.resources.requests.memory=500Mi \
+      --set scanner.resources.requests.cpu=500m \
+      --set scanner.resources.limits.memory=2500Mi \
+      --set scanner.resources.limits.cpu=2000m
+fi
+
+kubectl -n stackrox rollout status deploy/central --timeout=3m
+
+echo "Forwarding ports to central"
+
+kubectl -n stackrox port-forward deploy/central --pod-running-timeout=1m0s 8000:8443 > /dev/null 2>&1 &
+
+echo "Generating an init bundle with shared secrets"
+
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+  --password "${STACKROX_ADMIN_PASSWORD}" \
+  central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
+
+installflags=("-f stackrox-init-bundle.yaml" "--set clusterName=\"my-secured-cluster\"")
```

Comment:
Does this work? I thought that the `-f` should be a separate array item to its value `stackrox-init-bundle.yaml`

How about this version?

```suggestion
installflags=('-f' 'stackrox-init-bundle.yaml' '--set' 'clusterName="my-secured-cluster"')
```

Reply:
It works yes, I concatenate the contents of `installflags` to the end of my helm install command, but perhaps that was bad form. I can refactor the flags as you suggest.

Reply:
yes, please - let's refactor this